src/pages/workspace/upgrade/UpgradeIntro.tsx

```diff
@@ -27,6 +27,7 @@ function UpgradeIntro({feature, onUpgrade, buttonDisabled, loading}: Props) {
     const {isExtraSmallScreenWidth, isSmallScreenWidth} = useResponsiveLayout();
     const {translate} = useLocalize();
     const iconSrc = feature.icon in Illustrations ? Illustrations[feature.icon as keyof typeof Illustrations] : Expensicon[feature.icon as keyof typeof Expensicon];
+    const iconAdditionalStyles = feature === CONST.UPGRADE_FEATURE_INTRO_MAPPING.approvals ? styles.br0 : undefined;
```

Comment:
Is this necessary? If so the comparison should be based on `feature.id`, otherwise please remove it

Reply:
It's necessary to check it so I'm leaving it with your changes :) fyi: since passed argument is enforced to be entirely one of the `UPGRADE_FEATURES`, I thought that it is pretty much the same as comparing them over `id`, why does it matter here?

Reply:
Objects are compared by reference, they will be equal only if they refer to the same memory address which is not the case.

```js
const approvals = {id: "approvals"};
approvals === {id: "approvals"}; // false
```